go/obscuronode/enclave/main/cli.go

```diff
@@ -6,24 +6,30 @@ import (
 
 const (
 	// Flag names, defaults and usages.
-	nodeIDName    = "nodeAddress"
-	nodeIDDefault = ""
-	nodeIDUsage   = "The 20 bytes of the node's address (default \"\")"
+	nodeIDName    = "nodeID"
+	nodeIDDefault = 1
+	nodeIDUsage   = "A integer representing the 20 bytes of the node's address (default 1)"
 
-	portName    = "port"
-	portDefault = 11000
-	portUsage   = "The port on which to serve the Obscuro enclave service"
+	addressName    = "address"
+	addressDefault = ":11000"
```

Comment:
Yeah. If you specify `localhost:11000`, Docker does not expose the port (since it binds its own localhost). This syntax works outside Docker, but also ensures that it is exposed by the Docker image.

Reply:
Should we create syntaxs specific for Docker ?

Binding to nil: is the same as binding to 0.0.0.0 ? I wonder if this is the same across all OSes 🤔

Reply:
Ok - I'll use `localhost:11000` as the default, and specify in the Docker readme to override the address.